code/modules/projectiles/guns/projectile/automatic/sts35.dm

```diff
@@ -1,6 +1,8 @@
 /obj/item/weapon/gun/projectile/automatic/sts35
-	name = "STS-35"
-	desc = "The rugged STS-35 is a durable automatic weapon, popular on frontier worlds. Uses .30 Rifle rounds. This one is unmarked."
+	name = "OR SDF \"STS-35\""
+	desc = "The rugged STS-35 is a durable automatic weapon, made by Oberth Republic Self Defence Force. \
+			Extremely efficient rifle that was put in service right before collapse of the Republic, this weapon can be found almost anywhere in the galaxy by now. \
```

Comment:
Use "An extremely efficient rifle design" or "Extremely efficient rifle design". As extremely efficient rifle makes it sound like there's only one of them out there.